assembl/static2/js/app/components/debate/common/toggleCommentButton.jsx

```diff
@@ -2,12 +2,21 @@
 import React from 'react';
 import { Link } from 'react-router';
 
-// TODO: <span className="assembl-icon-up-open" />
+export type ToggleCommentButtonProps = {
+  /** Expand flag that changes the icon style */
+  isExpanded: boolean,
+  /** On click callback function */
+  onClickCallback: Function
+};
 
-const ToggleCommentButton = () => (
-  <Link className="action-toggle">
-    <span className="assembl-icon-down-open" />
-  </Link>
-);
+const ToggleCommentButton = ({ isExpanded, onClickCallback }: ToggleCommentButtonProps) => {
+  const iconStyle = isExpanded ? 'up' : 'down';
```

Comment:
use classnames

Reply:
@amensouissi is it worth it to import it when you're just dealing with 1 class depending on a boolean?

Reply:
same comment as @MilanBarande